Services/CO.CDP.EntityVerification.Tests/Events/EventDeserializerTest.cs

```diff
@@ -10,6 +10,18 @@ public class EventDeserializerTest
     [Fact]
     public void ItDeserializesOrganisationRegisteredEvent()
     {
+        var organisationRegistered1 = new OrganisationRegistered()
+        {
+            Id = Guid.NewGuid(),
+            Name = "name",
+            Identifier = new Identifier() { Id = "1234", LegalName = "name", Scheme = "test" },
+            Roles = new List<string>(),
+            AdditionalIdentifiers = new List<Identifier>()
+        };
+
+        string s = JsonSerializer.Serialize(organisationRegistered1);
```

Comment:
Looks like this isn't asserted on anywhere. Why create a new value? just alter the one below `organisationRegistered`.